dags/traffic_transfer.py

```diff
@@ -24,20 +36,30 @@
 for name in names:
     list_names.append(slack_ids.get(name, '@Unknown Slack ID')) #find slack ids w/default = Unkown
 
-def task_fail_nattery_slack_alert(context):
+def task_fail_slack_alert(context):
     slack_webhook_token = BaseHook.get_connection(SLACK_CONN_ID).password
-    task_msg = 'The {task} in updating traffic failed, {slack_name} go fix it meow :meow_notlike: '.format(
+    slack_msg = """
+            :red_circle: Task Failed / Tâche échouée.
+            {slack_name} please check.
+            *Task*: {task}
+            *Dag*: {dag}
+            *Execution Time*: {exec_date}
+            *Log Url*: {log_url}
+            """.format(
+            slack_name=' '.join(list_names),
             task=context.get('task_instance').task_id,
-            slack_name = ' '.join(list_names))
-        
-    slack_msg = task_msg + """(<{log_url}|log>)""".format(
-            log_url=context.get('task_instance').log_url,)
```

Comment:
Can you revert the Slack Alert message format?

Reply:
@radumas done!